### DISCUSSION TITLE
How to i uninstall

### DISCUSSION THREAD

**Discussion by estonianfarmer** [piotrulos/MSCModLoader#191] [Help me] [answered]
So, the problem im facing is i cant uninstall the loader. i wanna uninstall it because its not for me, but the button for it doesnt work. Deleting all MSCLOADER files either dont work. What do i do?

**Comment by estonianfarmer** [comment]
![Uploading image.png…]()

**Comment by piotrulos** [comment]
make sure you are uninstalling it from correct path

**Reply by estonianfarmer** [reply]
i am doing it all correctly but i dont know what the issue is, the remove button is unavailable

**Reply by piotrulos** [reply]
click install again and then click remove, if you removed some files manually it may not detect that it's installed.

**Reply by estonianfarmer** [reply]
i also cant install it again, im really running out of ideas what to do, the files are back tho

**Reply by piotrulos** [reply]
did you select the msc folder?

**Reply by estonianfarmer** [reply]
yes i did

**Reply by piotrulos** [reply]
show the log file

**Reply by estonianfarmer** [reply]
[Log.txt](https://github.com/piotrulos/MSCModLoader/files/14158474/Log.txt)

**Reply by piotrulos** [reply, marked as answer]
try deleting "doorstop_config.ini" from game folder

**Reply by estonianfarmer** [reply]
give me a sec

**Reply by estonianfarmer** [reply]
it does seem like nothing exists anymore, so i think its fixed? i cant see the missing mod errors neither anything associated with MSCLOADER
